packages/strapi-design-system/src/Button/utils.js

```diff
@@ -48,6 +48,13 @@ export const getHoverStyle = ({ theme, variant }) => {
     `;
   }
 
+  if (variant === 'default') {
```

Comment:
I think we'll have to turn these strings into constants at some points.

Reply:
I could try by adding a constants file in utils?